### ISSUE TITLE
how to fine-tune to improve the result of automatic_mask_generator_example.ipynb

### ISSUE THREAD

**Issue by githublqs** [facebookresearch/segment-anything#358]
in this https://encord.com/blog/learn-how-to-fine-tune-the-segment-anything-model-sam/ it say 
[Finally, we can generate the masks. Note that here we are in single mask generation mode (in contrast to the 3 masks that are normally output).]
with torch.no_grad():
      sparse_embeddings, dense_embeddings = sam_model.prompt_encoder(
          points=None,
          boxes=box_torch,
          masks=None,
      )
Does it mean that the parameter masks=False must be specified during fine-tune?
and parameter boxes must be specified ?
